app/models/neighbour.rb

```diff
@@ -18,6 +18,8 @@ class AddressValidationError < StandardError; end
 
   validate :validate_address_format
 
+  before_save :update_latlong, if: :address_changed?
```

Comment:
Think it would be useful to add a model spec when the address is added/updated that the `NeighbourLatLongUpdateJob` is called